Code/Editor/EditorPreferencesPageViewportCamera.cpp

```diff
@@ -185,11 +186,19 @@ void CEditorPreferencesPage_ViewportCamera::CameraMovementSettings::Reflect(AZ::
                 "Default Orbit Distance",
                 "The default distance to orbit about when there is no entity selected")
             ->Attribute(AZ::Edit::Attributes::Min, minValue)
+            ->DataElement(
+                AZ::Edit::UIHandlers::SpinBox,
+                &CameraMovementSettings::m_gotoPositionDuration,
+                "Camera Go To Position Duration",
+                "Time it takes for the camera to interpolate to a given position")
+            ->Attribute(AZ::Edit::Attributes::Visibility, &CameraMovementSettings::GoToPositionDurationVisibility)
+            ->Attribute(AZ::Edit::Attributes::Min, minValue)
             ->DataElement(
                 AZ::Edit::UIHandlers::CheckBox,
                 &CameraMovementSettings::m_goToPositionInstantly,
                 "Camera Go To Position Instantly",
                 "Camera will instantly go to the set position and won't interpolate there")
+            ->Attribute(AZ::Edit::Attributes::ChangeNotify, AZ::Edit::PropertyRefreshLevels::EntireTree)
```

Comment:
Suggestion: It would be nice to add a comment here to explain why the EntireTree refresh is needed. (I'm assuming it's to refresh the visibility of the Go To Position Duration?

Suggestion: If you make the duration readonly instead of hidden, it might be a little bit less jarring of a change (items won't reposition on the screen) and as a bonus you only have to refresh AttributesAndValues, not EntireTree.